packages/core/content-type-builder/admin/src/components/ListRow/index.js

```diff
@@ -38,7 +39,7 @@ function ListRow({
   const isMorph = type === 'relation' && relation.includes('morph');
   const ico = ['integer', 'biginteger', 'float', 'decimal'].includes(type) ? 'number' : type;
 
-  let readableType = type;
+  let readableType = customField || type;
```

Comment:
The readableType is just use for the translation so why not keeping the logic and just create a new component instead?

Something like the following:

```
const DisplayedType = ({ customField, readableType, repeatable }) => {
 if (customField) { 
   return <Typography>My custom field trad</Typography>
}

 return (
 <Typography>
            {formatMessage({
              id: getTrad(`attribute.${readableType}`),
              defaultMessage: type,
            })}
            &nbsp;
            {repeatable &&
              formatMessage({
                id: getTrad('component.repeatable'),
                defaultMessage: '(repeatable)',
              })}
          </Typography>
)
```

Reply:
Sounds great!